docs/architecture/tokens.md

```diff
@@ -5,32 +5,35 @@ last_update:
   date: 28 November 2023
 ---
 
-In a blockchain ecosystem, a digital asset that can be transferred between accounts is called a _token_.
-Tezos supports many types of tokens, including:
+Any blockchain ecosystem relies on a native token that can be transferred between accounts and in which transaction fees are paid in.
+The native token of Tezos is tez (also known as XTZ or represented by the symbol ꜩ).
+
+But other tokens representing some value in digital for can be programmed in a blockchain, for instance using smart contracts.
+Tokens fall in two broad categories:
+
+- Fungible tokens, whose instances are interchangeable and represent the same value,
+- Non-fungible tokens (NFTs), which are unique digital assets that model the ownership of some digital or real object
+
+Many types of fungible tokens are already implemented in Tezos, including:
 
-- The native token tez (also known as XTZ or represented by the symbol ꜩ), which is the primary token on Tezos and the token that transaction fees are paid in
-- Fungible tokens, which are interchangeable and accounts can own any amount of, like tez
-- Non-fungible tokens (NFTs), which are unique digital assets that can represent ownership of something
 - Stablecoins, which are tied to the price of fiat currencies such as USD and EUR
 - Wrapped tokens, which represent tokens from another blockchain or another standard; see [Wrapped tokens](#wrapped-tokens)
 
-It's important to remember that in most cases, Tezos tokens are managed by smart contracts.
-Tokens are not stored directly in accounts; instead, smart contracts keep a ledger of how many tokens different accounts hold.
-One exception is tickets, which are directly stored and managed by smart contracts.
+Tezos is also reputed as a platform for owning and exchanging various types of NFTs.
 
-To learn about tokens, see these tutorials:
+In most cases, (non-native) tokens are managed by smart contracts.
+They are not stored directly in accounts; instead, smart contracts keep a ledger of how many tokens are hold by different accounts.
```

Comment:
```suggestion
They are not stored directly in accounts; instead, smart contracts keep a ledger of how many tokens each account holds.
```
Making this active to be a little more direct.

Reply:
Applied, thanks.